File: phaseGateOne/java/TestStudentGrade.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class TestStudentGrade{
  @Test
  public void testThatEverageReturnsCorrectResult(){
    int[][] studentScores = {{1,2,3,1,2,3},{1,2,3,4,5}};
    double actual = StudentGrade.getAverageOf(studentScores[0]);
    double expected = 2.0;
    assertEquals(expected, actual);
  }
  
  @Test
  public void testThatgetTotalScoresReturnsCorrectResult(){
    int[][] studentScores = {{1,2,3,1,2,3},{1,2,3,4,5}};
    int[] actual = StudentGrade.getTotalScores(studentScores);
    int[] expected = {12, 15};
    assertArrayEquals(expected, actual);
  }
}

File: phaseGateOne/java/test_studentgrade.py
from unittest import TestCase
from studentgrades import *

class TestStudentGrade(TestCase):
    def test_that_collect_scores_function_exists(self):
        collect_scores(1,1)
        
    def test_that_everage_returns_correct_result(self):
        actual = get_average_score_of([1,2,3])
        expected = 2.0
        self.assertEquals(actual, expected)
